content/docs/dvclive/api-reference/live/index.md

```diff
@@ -25,9 +25,13 @@ live = Live()
 A `Live()` instance is required in order to log machine learning metrics and
 other metadata.
 
-⚠️ `Live()` will remove all existing DVCLive related files under `path` unless
+<admon type="warn">
+
+`Live()` will remove all existing DVCLive related files under `path` unless
```

Comment:
Does Prettier complain if you don't leave a blank line?

Reply:
I guess the code is easier to read and we do this with `<details>` as well. Up to you